src/components/shared/MarkerTooltipContents.js

```diff
@@ -11,6 +11,19 @@ import formatTimeLength from '../../utils/format-time-length';
 import type { TracingMarker } from '../../types/profile-derived';
 import type { MarkerPayload } from '../../types/markers';
 
+function markerDetail(key, label, value) {
```

Comment:
```js
function _markerDetail(key: string, label: string, value: string) {
  ...
}
```

We generally add types to all of the function signatures so they can be covered by Flow. And we also generally use `_functionName` for non-exported functions.